mrp_production_category_notes/tests/test_mrp_produccion_category_notes.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2025 Angel Rivas <angel.rivas@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestMrpProduccionCategoryNotes(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.product_category_with_mrp_notes = cls.env["product.category"].create(
+            {
+                "name": "Test Category with Mrp Notes",
+                "mrp_notes": "<table><tr><td>Test</td></tr></table>",
+            }
+        )
+
+        cls.product_category_without_mrp_notes = cls.env["product.category"].create(
+            {
+                "name": "Test Category without Mrp Notes",
+                "mrp_notes": "",
+            }
+        )
+
+        cls.product_with_mrp_notes = cls.env["product.product"].create(
+            {
+                "name": "Test Product with Mrp notes",
+                "categ_id": cls.product_category_with_mrp_notes.id,
+            }
+        )
+
+        cls.product_without_mrp_notes = cls.env["product.product"].create(
+            {
+                "name": "Test Product without Mrp notes",
+                "categ_id": cls.product_category_without_mrp_notes.id,
+            }
+        )
+
+        cls.uom_id = cls.product_with_mrp_notes.uom_id.id
+
+        cls.mrp_production = cls.env["mrp.production"].create(
+            {
+                "product_id": cls.product_with_mrp_notes.id,
+                "product_uom_id": cls.uom_id,
+            }
+        )
+
+    def test_get_category_notes(self):
+        table_html = self.mrp_production._get_category_notes(
```

Comment:
When moving the _get_category_notes method to product, this test will have to be modified.